### PR TITLE
fix: save adblock hosts at compile time

### PR DESCRIPTION
**Pull request by dylantientcheu**
### What
Saving adblocks host list file at compile time.

[DEX-1437](https://algolia.atlassian.net/browse/DEX-1437)

### Why
We previously fetched the raw file from Github at each Renderscript start, now we fetch it at build, keep the file and read it from fs (filesystem).

Related to:
#880 

[DEX-1437]: https://algolia.atlassian.net/browse/DEX-1437?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,4 +92,4 @@
     "@semantic-release/npm": "10.0.6"
   },
   "packageManager": "yarn@4.0.2"
-}
+}
```
Comment (sbellone) [inline package.json:95]: Nit: If you can configure your editor to keep/add those final `\n` :pray: 
```suggestion
}

```

**Review by sbellone — APPROVED** [review]
:rocket:

**Review by clemfromspace — DISMISSED** [review]
🚀

File: scripts/update_adblock_hosts.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+TARGET_DIR="./dist/lib/browser"
+TARGET_FILE="adblock_hosts.txt"
+
+if curl -o "${TARGET_DIR}/${TARGET_FILE}" "$URL" -s; then
+  echo "✅ adblock hosts download successful."
+else
+  echo "❌ adblock hosts download failed."
+  exit 1
+fi
```
Comment (sbellone) [inline scripts/update_adblock_hosts.sh:14]: The comment was for all of them :grimacing: :smiley: 
```suggestion
fi

```

**Review by sbellone — APPROVED** [review]

**Review by sbellone — APPROVED** [review]
:raised_hands:

**Pull request merged**